backend/src/main/repository/PromotionRepository.ts

```diff
@@ -50,10 +50,14 @@ export class PromotionRepository extends Repository<Promotion> {
       });
     }
 
+    // see https://github.com/ubclaunchpad/foodies/issues/54
     if (promotionQuery?.expirationDate) {
-      queryBuilder.andWhere('promotion.expirationDate >= :date', {
-        date: promotionQuery.expirationDate,
-      });
+      queryBuilder.andWhere(
+        "promotion.expirationDate ::timestamptz at time zone 'UTC' >= :date ::timestamptz at time zone 'UTC'",
```

Comment:
Sorry if you explained this in the past and I'm forgetting but what's the purpose of this?

Reply:
Oh this is if we want to filter promotions by expiration date, so for example if a user wants to find all promotions that are not expired after a given date. Does that answer your question?

Reply:
Oooh I see, yup that does!